yb-voyager/cmd/import.go

```diff
@@ -71,6 +72,10 @@ func validateImportFlags() {
 	validateImportObjectsFlag(target.ImportObjects, "object-list")
 	validateImportObjectsFlag(target.ExcludeImportObjects, "exclude-object-list")
 	validateTargetSchemaFlag()
+	// For beta2.0 release (and onwards until further notice)
+	if disableTransactionalWrites {
+		fmt.Println("WARNING: --disable-transactional-writes is an experimental feature")
```

Comment:
nit: fmt.Println("WARNING: --disable-transactional-writes feature is in the experimental phase, not for production use case.")